srcpkgs/haskell-language-server/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'haskell-language-server'
+pkgname=haskell-language-server
+version=1.0.0
+revision=1
+# GHC only panics on i686
+# https://github.com/void-linux/void-packages/pull/27961/checks?check_run_id=2181519625#step:8:5936
+archs="~i686"
+build_style="haskell-stack"
+make_build_args="--stack-yaml stack-8.8.4.yaml"
+makedepends="ncurses-devel ncurses-libtinfo-devel icu-devel zlib-devel"
+short_desc="Haskell Language Server: Integration of ghcide and haskell-ide-engine"
```

Comment:
Just "Integration point for ghcide and haskell-ide-engine", noneed to repeat package name.

Reply:
ping